src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+.card {
+  box-sizing: border-box;
+  width: $cardwidth;
+  height: 408px;
```

Comment:
do not hardcode card height. It is not a good idea to hardcode elements with content inside